package.json

```diff
@@ -197,7 +197,8 @@
             ".kit": "kitc --run",
             ".v": "v run",
             ".vsh": "v run",
-            ".sass": "sass --style expanded"
+            ".sass": "sass --style expanded",
+            ".cu": "cd $dir && nvcc $fileName -o $fileNameWithoutExt && ./$fileNameWithoutExt"
```

Comment:
On Windows, the `./$fileNameWithoutExt` would not work in CMD.

![image](https://user-images.githubusercontent.com/1050213/76176000-04439180-61ea-11ea-85b7-4028b0dce1bc.png)

So we may need to use `$dir$fileNameWithoutExt`.

Please help test it on macOS, Linux and Windows. Thanks a lot!

Reply:
Right. I do not have a macOS to test this, I don't even know if its common, as not current Mac comes with Nvidia GPU that is mandatory for CUDA programming.
For Windows I'm trying to test it, but CUDA is not officially supported on vscode for Windows, that's why I only tried on Linux. It needs Visual Studio IDE installed to work apparently.
I'll install it and try, but you suggestions seems to work. I did not remembered that for windows its .\ and for Linux its ./

Reply:
Ok, just tried. It works `$dir$fileNameWithoutExt` on both Linux and Mac